beacon_node/eth2_libp2p/src/rpc/protocol.rs

```diff
@@ -28,16 +28,16 @@ use types::{BeaconBlock, EthSpec, Hash256, MainnetEthSpec, Signature, SignedBeac
 lazy_static! {
     // Note: Hardcoding the `EthSpec` type for `SignedBeaconBlock` as min/max values is
     // same across different `EthSpec` implementations.
-    pub static ref SIGNED_BEACON_BLOCK_MIN: usize = SignedBeaconBlock::<MainnetEthSpec> {
-        message: BeaconBlock::empty(&MainnetEthSpec::default_spec()),
-        signature: Signature::empty(),
-    }
+    pub static ref SIGNED_BEACON_BLOCK_MIN: usize = SignedBeaconBlock::<MainnetEthSpec>::from_block(
+        BeaconBlock::<MainnetEthSpec>::empty(&MainnetEthSpec::default_spec()),
```

Comment:
```suggestion
        BeaconBlock::empty(&MainnetEthSpec::default_spec()),
```

I checked this locally and I'm confident it'll compile.